pkg/controller/volume/scheduling/scheduler_assume_cache_test.go

```diff
@@ -109,13 +109,13 @@ func TestAssumePV(t *testing.T) {
 
 	for name, scenario := range scenarios {
 		cache := NewPVAssumeCache(nil)
-		internal_cache, ok := cache.(*pvAssumeCache)
+		internalCache, ok := cache.(*pvAssumeCache)
```

Comment:
` don't use underscores in Go names; var internal_cache should be internalCache`